tests/check_odov3.sh

```diff
@@ -49,6 +49,7 @@ ginkgo run --procs 2 \
   --skip="stack: java-vertx version: 1.4.0 starter: vertx-istio-security-booster" \
   --skip="stack: java-vertx version: 1.4.0 starter: vertx-messaging-work-queue-booster" \
   --skip="stack: java-websphereliberty-gradle version: 0.4.0 starter: rest" \
+  --skip="stack: jhipster-online version: 2.23.0 starter: jhipster-online" \
```

Comment:
Your stack version might be a typo:
```suggestion
  --skip="stack: jhipster-online version: 2.33.0 starter: jhipster-online" \
```

Reply:
Thanks Michael, I send the change.